debug_test.go

```diff
@@ -30,6 +30,8 @@ import (
 )
 
 func TestJSON(t *testing.T) {
+	t.Parallel()
```

Comment:
Parallelizing everything. [There will be an extra comment in this PR for a bugfix in a test]